website/blog/2024-04-15-0.74-post.md

```diff
@@ -0,0 +1,407 @@
+---
+title: 'React Native 0.74'
+authors:
+  [hurali, alanjhughes, alfonsocj, ingridwang, huntie, cortinico]
+tags: [release]
+date: 2024-04-15
+---
+
+Today we're releasing React Native 0.74!...
+
+## Highlights
+
+<!-- https://github.com/facebook/react-native/blob/main/CHANGELOG.md -->
+
+- [React Native Frameworks](/blog/2024/04/15/0.74-post#react-native-frameworks)
+- New layout behaviors
+- Support for `align-content: 'space-evenly'`
+- New Architecture support for `position: 'static'`
```

Comment:
I thought we decide to put all the New Architecture updates together in the working group.
Am I remembering it wrong? Or is there something special related to this update that we want to call it out in the blogpost?